DESCRIPTION

```diff
@@ -19,13 +19,15 @@ LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 Imports: 
-    EML
+    EML,
+    utils
```

Comment:
I was playing around with the vignette because it was not building properly. As part of that process I was able to find a solution that had utils as a dependency. However, I found a better solution and then forgot to remove utils.

Reply:
Now it is removed.